services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/entity/VpcState.java

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.subnet.entity;
+
+import com.futurewei.alcor.common.entity.CustomerResource;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class VpcState extends CustomerResource {
```

Comment:
Do you think it will be a good idea to use VpcState and VpcStateJson under /web folder, or replicate one copy in the microservice that needs that?

Reply:
I used to replicate one copy in the microservice that needs that, but we can talk about it ~